services/mock-service/src/fsp-integration/safaricom/safaricom.mock.service.ts

```diff
@@ -133,21 +139,39 @@ export class SafaricomMockService {
         },
       },
     };
+    const timeoutCallbackResponse = {
```

Comment:
isn't the point that the callback response is exactly the original request, so just 'const timeoutCallbackResponse = transferDto;'?